README.md

```diff
@@ -77,7 +77,7 @@ Star us, and you will receive all releases notifications from GitHub without any
 
 - **Hyper merged** — Write, draw and plan all at once. Assemble any blocks you love on any canvas you like to enjoy seamless transitions between workflows with AFFiNE.
 - **Privacy focussed** — AFFiNE is built with your privacy in mind and is one of our key concerns. We want you to keep control of your data, allowing you to store it as you like, where you like while still being able to freely edit and view your data on-demand.
-- **Offline-first** - With your privacy in mind we also decided to go offline-first. This means that AFFiNE can be used offline, whether you want to view or edit, with support for conflict-free merging when you are back online.
+- **Offline-first** — With your privacy in mind we also decided to go offline-first. This means that AFFiNE can be used offline, whether you want to view or edit, with support for conflict-free merging when you are back online.
```

Comment:
why use `—` instead of `-`

Reply:
As all the others topics used `—` instead of `-`, I thought that this was a typo, am I wrong? Does exist a reason to use `-` instead of `—` in that specific topic?

Reply:
fun fact, I didn't notice that

Reply:
Reading the file is difficult to notice that but reading the README.md in the root the `-` is screaming hahaha